content/WebApiHost/Startup.cs

```diff
@@ -65,14 +64,13 @@ public void ConfigureServices(IServiceCollection services)
 
             // Configures the NybusHostedService so that Nybus is started when the web application is started
             services.AddHostedService<NybusHostedService>();
-//#endif
-//#if (AddWcfDiscovery)
+            //#endif
 
-            // Configures the WCF Discovery from the current configuration
-            services.AddDiscovery(Configuration);
-            services.AddSingleton<IDiscoveryService, NetTcpDiscoveryService>();
-//#endif
-//#if (AddNybusBridge || ConfigureAWS)
+            //#if (AddDiscoveryAdapter)
+            services.ConfigureServiceDiscovery(Configuration.GetSection("Discovery"));
+            services.AddServiceDiscoveryAdapter();
+            //#endif
+            //#if (AddNybusBridge || ConfigureAWS)
```

Comment:
here I would also include this code

```
services.ConfigureServiceDiscovery(o =>
        {
            o.ConfigureDiscoveryAdapterBinding = binding =>
            {
                binding.Security.Mode = SecurityMode.None;
            };
        });

services.AddBindingCustomization(binding => binding.Security.Mode = SecurityMode.None);
```

Reply:
Also will include this code, but commented because we don't have the service reference in this template project. like this 

`//services.DiscoverServiceUsingAdapter<IDiscoverableService>();`

where IDiscoverableService is the interface of the WCF service which we need to discover and replace with correct one after we now which WCF service we need to use in the project